std/core/memory.whi

```diff
@@ -0,0 +1,62 @@
+export inline fun u8_load(ptr: i32): u8 {
+    return #(wasm) {
+        local.get $ptr
+        i32.load8_u
+    }
+}
+
+export inline fun i8_load(ptr: i32): i8 {
+    return #(wasm) {
+        local.get $ptr
+        i32.load8_s
+    }
+}
+
+export inline fun u16_load(ptr: i32): u16 {
+    return #(wasm) {
+        local.get $ptr
+        i32.load16_u
+    }
+}
+
+export inline fun i16_load(ptr: i32): i16 {
+    return #(wasm) {
+        local.get $ptr
+        i32.load16_s
+    }
+}
+
+export inline fun u32_load(ptr: i32): u32 {
+    return #(wasm) {
+        local.get $ptr
+        i64.load32_u
+    }
+}
+
+export inline fun i32_load(ptr: i32): i32 {
+    return #(wasm) {
+        local.get $ptr
+        i32.load
+    }
+}
+
+export inline fun i64_load(ptr: i32): i64 {
```

Comment:
i64 load would be nice too, it should be the same but transmuted to the u64 type

Reply:
I assume you mean u64. Would it be fine to use `i64.load32_u` twice and thus return `[i64, i64]` as two compliments or should I just set return type as `u64` and it will be handled by the compiler?

Reply:
just set the return type as `u64`, all arithmetic and shit is handled by the compiler